transform_to_deno.js

```diff
@@ -0,0 +1,74 @@
+const fs = require('fs').promises;
+const path = require('path');
+
+const [,, srcDir = 'lib', destDir = 'lib/deno', extension = '.d.ts'] = process.argv;
+
+const packageJson = require('./package.json');
+
+const importMap = {};
+
+// fill the import map with data from package.json
+for (const [dep, version] of Object.entries(packageJson.dependencies)) {
+    importMap[path.basename(dep)] = `https://cdn.skypack.dev/${dep}@${version}?dts`;
```

Comment:
Shouldn't it be `...{version}?d.ts` ?

Reply:
nope :D
?dts is a query param for skypack to enable deno typescript compatibility
[(docs)](https://docs.skypack.dev/skypack-cdn/code/deno#automatic-typescript-declarations)

Reply:
Allright thanks :)